network/stormshield/snmp/mode/hanodes.pm

```diff
@@ -152,15 +158,15 @@ sub new {
                                 "filter-node:s"         => { name => 'filter_node' },
                                 "percent"               => { name => 'percent' },
                                 "warning-state:s"       => { name => 'warning_state', default => '' },
-                                "critical-state:s"      => { name => 'critical_state', default => '%{state} eq "offline"' },
+                                "critical-state:s"      => { name => 'critical_state', default => '%{state} =~ /offline/i' },
                                 });
     return $self;
 }
 
 sub change_macros {
     my ($self, %options) = @_;
 
-    foreach (('warning_state', 'critical_state')) {
+    foreach ('warning_state', 'critical_state') {
```

Comment:
@cgagnaire think that double parenthesis is here for backward compatibility. Did you test this on CentOS 6 perl version ?

Reply:
Ok just test and it works. Do not know why Quentin used to write it like this as it's not a know convention, no reason so let merge this one.